libs/cart/src/drivers/interfaces/transformers/cart-transformer.interface.ts

```diff
@@ -0,0 +1,16 @@
+import { DaffCart } from '../../../models/cart';
+import { InjectionToken } from '@angular/core';
+
+/**
+ * An interface that must be implemented by cart transformer services.
+ * 
+ * This transformer is responsible for transforming the returned
+ * cart from a given platform into a Generic DaffCart.
+ */
+export interface DaffCartTransformerInterface<RES, T extends DaffCart> {
+  transform(response: RES): T;
```

Comment:
What's your reason for using a generic `transform` parameter for just this transformer?

Reply:
No specific reason. I'm still a bit uncertain about typing transformers at the moment.

Reply:
I think typing arguments is probably useless, and we should be only typing the return types.